docker/docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+cygnus:
+    image: tidchile/fiware-cygnus
```

Comment:
What does this path means? Is it about the location of the contained at dockerhub? In that case, this must be "telefonicaid/fiware-cygnus".